playbooks/roles/airship-deploy-ucp/defaults/main.yml

```diff
@@ -1,4 +1,9 @@
 ---
+ucp_namespace_name: "{{ ucp_namespace | default('ucp') }}"
```

Comment:
namespace variables are already defined in common-vars.yaml.

Reply:
Oddly, only ucp namespace is defined. hmm, may be should define other 2 there as well. Will do that